core/src/trezor/ui/layouts/tt_v2/reset.py

```diff
@@ -1,10 +1,36 @@
 from typing import TYPE_CHECKING
 
 from trezor import wire
+from trezor.enums import BackupType, ButtonRequestType
+
+import trezorui2
+
+from ..common import interact
+from . import _RustLayout
```

Comment:
note: ISTM that most of this file should move (back) to `apps/recovery` because at this point the basic UI elements are things like `select_word`, `request_number` etc.

something to keep in mind for tomorrow